mantatail.py

```diff
@@ -133,6 +184,19 @@ def recv_loop(state: ServerState, user_host: str, user_socket: socket.socket) ->
 
 
 class UserConnection:
+    """
+    Represents the connection between server & client.
+    Starts a Queue on a separate thread on which the client receives all messages from the server.
+
+    Format examples:
+    - Nick: Alice
+    - User message: AliceUsr 0 * Alice's Real Name
+    - Username: AliceUsr
+    - User Mask Alice!AliceUsr@127.0.0.1 (Nick!Username@Host)
+
+    All references to UserConnection (lists, dicts, etc.) are based on Nick.
```

Comment:
This again expresses a simple thing using lots of complicated technical details.

```suggestion
    Usually the nick is used when referring to the user.
```